src/binder/bind_expression/bind_function_expression.cpp

```diff
@@ -68,6 +74,34 @@ static std::vector<LogicalType> getTypes(const expression_vector& exprs) {
     return result;
 }
 
+std::shared_ptr<Expression> ExpressionBinder::bindAliasFunction(
+    const ParsedExpression& parsedExpression, const expression_vector& children,
```

Comment:
Why this needs to be a separate function. Can u write a bindFunc for aliasFunction and generate `FunctionStringBindData` there? We should reuse existing code path as much as possible. Let me know if this is not doable.